pkgs/tools/graphics/sanjuuni/default.nix

```diff
@@ -4,17 +4,18 @@
 , pkg-config
 , ffmpeg
 , poco
+, opencl
```

Comment:
Maybe this https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/version-management/lucky-commit/default.nix#L23 is helpful but not sure

Reply:
I believe I've fixed it in the latest push.